TrelloAutotest/Tests/ManageBoardsTest.cs

```diff
@@ -0,0 +1,37 @@
+﻿using NUnit.Framework;
+using TrelloAutotest.Pages;
+using TrelloAutotests.Tests;
+
+namespace TrelloAutotest.Tests
+{
+    class ManageBoardsTest : BaseTest
+    {
+        private TrelloMainPage _trelloMainPage;
+        private TrelloCreateBoardPage _configureBoard;
+        private TrelloBoardPage _boardPage;
```

Comment:
All the above fields are unnecessary. They will have a sense if you initialize them in the constructor.